settings.py

```diff
@@ -13,10 +13,10 @@
     DB_URI = rds_config[0].get('credentials').get('uri')
 else:
     DB_HOST = os.getenv('SDX_STORE_POSTGRES_HOST', '0.0.0.0')
-    DB_PORT = os.getenv('SDX_STORE_POSTGRES_PORT', '5432')
-    DB_NAME = os.getenv('SDX_STORE_POSTGRES_NAME', 'postgres')
-    DB_USER = os.getenv('SDX_STORE_POSTGRES_USER', 'postgres')
-    DB_PASSWORD = os.getenv('SDX_STORE_POSTGRES_PASSWORD', 'secret')
+    DB_PORT = os.getenv('SDX_STORE_POSTGRES_PORT', '5433')
```

Comment:
Why have the DB settings been changed?

Reply:
We should be consistent, the db.env in sdx-compose sets everything to 'sdx' on port 5432, but sdx-store and sdx-console sets to postgres/secret on 5432...

Reply:
I think I'll revert it, but it probably needs to be looked at.  I don't think any of the postgres instances we use use the postgres/secret/5432 config.  Prod and preprod have their own, and compose exposes postgres on sdx/sdx/5433